phpBB/language/en/posting.php

```diff
@@ -241,7 +241,7 @@
 	'SMILIES_ARE_ON'			=> 'Smilies are <em>ON</em>',
 	'STICKY_ANNOUNCE_TIME_LIMIT'=> 'Sticky/Announcement time limit',
 	'STICK_TOPIC_FOR'			=> 'Stick topic for',
-	'STICK_TOPIC_FOR_EXPLAIN'	=> 'Enter 0 for a never ending Sticky/Announcement. Please note that this number is relative to the date of the post.',
+	'STICK_TOPIC_FOR_EXPLAIN'	=> 'Enter 0 for a never ending Sticky/Announcement/Global. Please note that this number is relative to the date of the post.',
```

Comment:
I waffled on it, but there are four choices given, Normal, Sticky, Announcement and Global. I've been around phpBB long enough to know that Global is a special case of Announcement, but I couldn't assume that every board admin would know that, so thought adding it would make it complete and clear.